tests/ssl-opt.sh

```diff
@@ -1622,10 +1622,11 @@ run_test() {
         return
     fi
 
-    print_name "$NAME"
-
     if [ "$LIST_TESTS" -gt 0 ]; then
+        print_name "${TEST_SUITE_NAME:-ssl-opt};$NAME"
```

Comment:
The files in `opt-testcases` are not actually different test suites: you can't invoke them separately. If we rearrange the `opt-testcases` files, it doesn't currently impact the reporting: it's just an implementation detail. So I don't think it makes sense to record their outcome separately.

Reply:
So checking the history, reporting the file name for `opt-testcases/*` was deliberate, done in https://github.com/Mbed-TLS/mbedtls/pull/5393. (By me, but I'd completely forgotten about it.) The goal at the time was to make outcome reporting consistent with outcome analysis. Since analysis was done by parsing file, it was easier to use the file name for analysis, and adapt reporting to use the file name.

Now that analysis is based on running the script, it's easier to use the executable script name for analysis, and adapt reporting back to using the executable name. And besides that, I think that since individual files in `opt-testcases` can't be run automatically, we should consider `ssl-opt` as a single “test suite”, so outcome reporting and outcome analysis should both use `ssl-opt`, as in 746bc0187b3f08e84bb33dc88007f05ff20ad7e5.